src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -129,9 +129,9 @@ export class GKEClusterSettingsComponent
   }
 
   ngOnDestroy(): void {
-    this.reset();
     this._unsubscribe.next();
     this._unsubscribe.complete();
+    this.reset();
```

Comment:
when reset. all the fields will be empty and yet not been unsubscribe so the api request for versions will be sent as empty  zones so the popup notification for wrong request will be shown so we have to unsubscribe first and then reset.

is there any  problem that been shown because of this change ?

Reply:
All forms related logic should stay above unsubscribe logic